extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/EnvVarHolder.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.quarkus.kubernetes.deployment;
+
+import static io.quarkus.kubernetes.spi.KubernetesEnvBuildItem.EnvType.configmap;
+import static io.quarkus.kubernetes.spi.KubernetesEnvBuildItem.EnvType.field;
+import static io.quarkus.kubernetes.spi.KubernetesEnvBuildItem.EnvType.secret;
+import static io.quarkus.kubernetes.spi.KubernetesEnvBuildItem.EnvType.var;
+
+import java.util.Collection;
+import java.util.Map;
+
+import io.quarkus.kubernetes.spi.KubernetesEnvBuildItem;
+
+/**
+ * @author <a href="claprun@redhat.com">Christophe Laprun</a>
```

Comment:
Please remove author tags, we don't use them in Quarkus

Reply:
I wondered about that but since I had seen some in some part of the code, I left it.

Reply:
Yeah, it's only left over from classes that have been around before the introduction of this policy